InternetHospital.UnitTests/Appointments/AppointmentServiceUnitTests.cs

```diff
@@ -223,6 +223,100 @@ private List<AppointmentForPatient> GetExpectedPatientAppointments(Appointment a
+                                           })
+                                           .With(a => a.UserId, PATIENT_ID)
+                                           .With(a => a.User, new User
+                                           {
+                                               Id = PATIENT_ID
+                                           })
+                                           .Create();
+
+            Mapper.Initialize(cfg => cfg.CreateMap<AppointmentSubscribeModel, Appointment>());
+
+            using (var context = new ApplicationContext(options))
+            {
+                context.Appointments.Add(fixtureAppointment);
+                context.SaveChanges();
+            }
+
+            using (var context = new ApplicationContext(options))
+            {
+                var appointmentService = new AppointmentService(context);
+
+                // act
+                var status = appointmentService.SubscribeForAppointment(fixtureAppointment.Id, patientId);
+
+                // assert
+                status.Should().BeTrue();
+            }
+        }
+
+        [Fact]
+        public void ShouldUnsubscribeForAppointment()
+        {
+            // arrange
+            const int ALLOWED_STATUS_ID = 2;
+            const int PATIENT_ID = 92;
+            var options = DbContextHelper.GetDbOptions(nameof(ShouldUnsubscribeForAppointment));
+            var fixture = FixtureHelper.CreateOmitOnRecursionFixture();
+
+            var fixturePatient = fixture.Build<User>()
+                .Create();
+
+            var patientId = fixturePatient.Id;
+
+            var fixtureAppointment = fixture.Build<Appointment>()
+                                           .With(a => a.StatusId, ALLOWED_STATUS_ID)
+                                           .With(a => a.Status, new AppointmentStatus
+                                           {
+                                               Id = ALLOWED_STATUS_ID
+                                           })
+                                           .With(a => a.UserId, PATIENT_ID)
+                                           .With(a => a.User, new User
+                                           {
+                                               Id = PATIENT_ID
+                                           })
+                                           .Create();
+
+            Mapper.Initialize(cfg => cfg.CreateMap<AppointmentUnsubscribeModel, Appointment>());
+
+            using (var context = new ApplicationContext(options))
+            {
+                context.Appointments.Add(fixtureAppointment);
+                context.SaveChanges();
+            }
+
+            using (var context = new ApplicationContext(options))
+            {
+                var appointmentService = new AppointmentService(context);
+
+                // act
+                var status = appointmentService.UnsubscribeForAppointment(fixtureAppointment.Id, patientId);
```

Comment:
Unsibscribe method doesnt use patientId inside, first of all rework service and make pull requst with changed method.

Reply:
why do you think that unsibscribe method doesnt use patientId inside?